content/docs/fundamentals/discovery@id.md

```diff
@@ -1,44 +1,52 @@
 ---
-$title: Membuat Laman Dapat Ditemukan
----
+$title: Membuat halaman Anda mudah ditemukan
+$order: 10
+$category: Deploy
 
+toc: true
+---
```

Comment:
Please remove all the added front matter from all of these files. Only the $title should be on the translated files.

Reply:
Updated front matter for all translation files for discovery.md.